includes/functions.php

```diff
@@ -68,10 +68,33 @@ function pods_cf_capture_entry($config, $form){
 		foreach ( $config['fields'] as $pod_field => $binding ) {
 			if ( ! empty( $binding ) ) {
 				$entry[ $pod_field ] = Caldera_Forms::get_field_data( $binding, $form );
-			}
 
-		}
+				// process file fields correctly
+				$real_pod_field = $pods->fields($pod_field);
+				if ($real_pod_field["type"] == "file") {
```

Comment:
@romci You should use `$file_field_types = PodsForm::file_field_types();` and then `if ( $real_pod_field && in_array( $real_pod_field['type'], $file_field_types, true ) )`